app/src/main/java/a3/com/convo/fragments/GameFragment.java

```diff
@@ -27,14 +28,21 @@
 
     private Context context;
     SwipeDeck cardStack;
+
+    // objectId of the other player
     private String friend;
+
     ParseUser player1;
     ArrayList<String> p1Likes;
     ParseUser player2;
     ArrayList<String> p2Likes;
     ArrayList<String> allLikes;
     CardAdapter adapter;
 
+    // Overall game timer elements
+    CountDownTimer timer;
```

Comment:
Do we need this timer as a member variable? Looks like we are using this only in onViewCreated